server/cassandraApp.js

```diff
@@ -0,0 +1,161 @@
+const express = require('express');
+const app = express();
+const cassie = require('../CassandraDB/index.js')
+const path = require('path');
+
+
+app.use(express.json())
+app.use('/products', express.static('./client/dist'))
+app.use('/', express.static('./client/dist'))
+
+
+// this function will find the highest id in the database that does not already have an entry. This 'highest' variable can be adjusted to narrow the
+// search down -- for example the database was explicitly seeded with 10,000,000 entries, so there's no need to look before that for unused ids
+let highest = 10000000;
+let searchForHighest = () => {
+  let tryHighest = cassie.execute(`SELECT * FROM mykea.items WHERE id = ${highest}`).then(results => results);
+
+  tryHighest
+  .then((results) => {
+    // console.log(highest) // for debugging this function
+    if (results.rows[0] !== undefined) {
+      highest++;
+      return searchForHighest();
+    }
+    console.log('highest reached at: ', highest)
+    return;
+  })
+};
+
+searchForHighest();
+
+app.get('/api/productOptions/products/:id', (req, res) => {
+
+  let constructedResponse;
+
+  let id = req.params.id;
+  let queryItems = `SELECT * FROM mykea.items WHERE id = ${id};`

```

Comment:
Looking at all these Cassandra queries makes me wonder how is cassandra now SQL if it uses the same data structure (tables) and the query language looks very similar (IF not the same)... I wonder if it's something that happens under the hood of Cassandra, like how it retrieves information and stuff.